skopt/tests/test_parameter.py

```diff
@@ -16,57 +18,74 @@ def check_distribution(Dist, vals, random_val):
 
 
 def test_distributions():
-    yield (check_distribution, Real, (1., 4.), 2.668088018810296)
-    yield (check_distribution, Real, (1, 4), 2.668088018810296)
+    a = Real(1, 25)
+    yield (check_distribution, Real, (1., 4.), 2.251066014107722)
+    yield (check_distribution, Real, (1, 4), 2.251066014107722)
     yield (check_distribution, Integer, (1, 4), 2)
     yield (check_distribution, Integer, (1., 4.), 2)
     yield (check_distribution, Categorical, ('a', 'b', 'c', 'd'), 'b')
     yield (check_distribution, Categorical, (1., 2., 3., 4.), 2.)
 
 
-def test_simple_grid():
-    expected = [(2, 1), (1, 2), (2, 2), (2, 1), (1, 2)]
+def check_distribution_limits(value, lower_bound, upper_bound):
+    assert_less_equal(lower_bound, value)
+    assert_greater_equal(upper_bound, value)
```

Comment:
`assert_greater` as the upper bound is not included. At least `np.alltrue(uniform(2, 12.-2).rvs(1000000)  < 12)` -> `True`